#
Header header
float64[] W
uint64[] dim_W
basis_fcn[] basis_fcn
time T #desired execution of the trajectory
